templates/cloud/public-cloud.html

```diff
@@ -1,170 +1,313 @@
+              }}              </a>
+            </h2>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              <a href="/pro">Ubuntu Pro</a> is a subscription that provides additional enterprise-grade support, security and compliance on top of Ubuntu LTS.
+            </p>
+            <hr class="is-muted" />
+            <ul class="p-list--divided">
+              <li class="p-list__item is-ticked">
+                10 years of maintenance and security updates for over 36,700 packages, up to 12 years with Legacy Support
+              </li>
+              <li class="p-list__item is-ticked">
+                Kernel security updates applied without reboot with <a href="/security/livepatch">Livepatch</a>
+              </li>
+              <li class="p-list__item is-ticked">
+                Hardening and audit tools for <a href="/security/certifications/docs/usg">CIS and DISA-STIG</a>, and <a href="/security/fips">FIPS certified components</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
     </div>
-  </div>
-</section>
+  </section>
 
-{% include "/shared/_ubuntu_pro_on_public_cloud.html" %}
-
-<section class="p-strip is-deep is-bordered">
-  <div class="u-fixed-width">
-    {% with title="A selection of our certified public cloud partners", json_feed_url="https://partners.ubuntu.com/partners.json?programme__name=Public%20Cloud&featured=true", feed_item_limit=10 %}{% include "shared/_partner-logos.html" %}{% endwith %}
-  </div>
-</section>
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr />
+      <h2 class="p-muted-heading">
+        A selection of our certified <a href="https://canonical.com/partners/public-cloud">public cloud partners&nbsp;&rsaquo;</a>
+      </h2>
+      {% with tittle="" , json_feed_url="https://partners.ubuntu.com/partners.json?programme__name=Public%20Cloud&featured=true", feed_item_limit=10 %}
+        {% include "shared/_partner-logos.html" %}
+      {% endwith %}
+    </div>
+  </section>
 
-<section class="p-strip is-bordered">
-  <div class="row p-divider">
-    <div class="col-6 p-divider__block">
-      <h2 class="p-heading--3">Fast service deployment with&nbsp;Juju</h2>
-      <div class="u-hide--medium u-hide--small u-align--center u-sv1">
-        {{
-          image(
-            url="https://assets.ubuntu.com/v1/162002c2-charms_juju.svg",
-            alt="Grid of juju charm icons including wordpress, hadoop, django, MySQL, MongoDB and OpenStack",
-            height="208",
-            width="504",
-            hi_def=True,
-            loading="lazy",
-          ) | safe
-        }}
+  <section class="p-section">
+    <div class="row">
+      <hr />
+      <div class="col-6 p-section--shallow">
+        <h2>
+          Select your
+          <br class="u-hide--large u-hide--medium" />
+          platform
+        </h2>
+      </div>
+      <div class="col-6">
```

Comment:
```suggestion
      <div class="col-9 col-start-large-4">
```
This works reasonably and doesn't compromise the design too much. What do you think @mattea-turic ?
![image](https://github.com/user-attachments/assets/322859f4-af25-4f4e-af01-843ac312e9d6)